openedx/features/philu_utils/sites.py

```diff
@@ -0,0 +1,23 @@
+"""
+All utilities for Django sites
+"""
+from logging import getLogger
+
+from django.contrib.sites.models import Site
+
+log = getLogger(__name__)
+
+
+def get_site(site_id):
+    """
+    Get site object from site id
+    Args:
+        site_id (int): the id of site
+
+    Returns:
+        site: The site object otherwise None
+    """
+    try:
+        return Site.objects.get(id=site_id)
+    except Site.DoesNotExist:
+        log.info('Site with id {id}, does not exists'.format(id=site_id))
```

Comment:
I guess we should log an error here, I also have concern about it, we are silently logging this info. In case this happens this function won't return anything. And it will break the code, can we some how tackle that. May be raising this exception again, but that will exactly be same as to not define this utility and use `.get` directly with exception handling.

Reply:
This function is created in `philu_utils` because it is not related to `openassessment` and it will return `None` in-case site id is wrong or does not exists, which is mentioned in the docstring. As per discussion, we are keeping it here as-is.

I have changes `log.info` to `log.error`.